app/app.R

```diff
@@ -79,6 +86,21 @@ ui <- page_navbar(
       )
     )
   ),
+  nav_panel(
+    "About",
+    layout_column_wrap(
+      width = 1/3,
+      heights_equal = "all",
+      fill = FALSE,
+      make_value_latest(data_full),
+      make_value_precip(data_full),
+      make_value_test(),
+    ),
+    card(
+      includeMarkdown("about.md"),
+      fill = TRUE #change this to TRUE to have the row of value boxes "frozen" to the top
+    )
+  ),
```

Comment:
To add a banner image you can do something like this:

``` r
  nav_panel(
    "About",
    card(img(src = "banner.jpg", align = "center"), fill = FALSE),
    layout_column_wrap(
      width = 1/3,
      heights_equal = "all",
      fill = FALSE,
      make_value_latest(data_full),
      make_value_precip(data_full),
      make_value_test(),
    ),
    card(
      includeMarkdown("about.md"),
      fill = FALSE 
    )
  ),
  ```
  
  Where `banner.jpg` is in a folder called `www` inside `app/`, so `app/www/banner.jpg`.  The image *must* be in `www/` for this to work.